# A single, two dimensional DoA measurement of an incoming sound
# recorded through a microphone array

uint32 sourceId   # Unique identifier of a sound source, used
                  # for tracking sound movements over time
                  
float32 angle     # Angle of arrivla of the sound, in radians